src/types/events/base-events.ts

```diff
@@ -160,7 +160,7 @@ export interface AppUninstalledEvent extends StringIndexed {
   type: 'app_uninstalled';
 }
 
-export interface CallsRejectedEvent extends StringIndexed {
+export interface CallRejectedEvent extends StringIndexed {
   type: 'calls_rejected';
```

Comment:
hmm, this line and https://github.com/slackapi/bolt-js/blob/f337fa2a582b797cc6b612c86dcf01759ce1ec4f/src/types/events/base-events.ts#L14 need to be changed

Reply:
sorry, i'm doing this too quickly and missing details. Thank you!

Reply:
No worries! ... but https://github.com/slackapi/bolt-js/pull/505#discussion_r431486330 is not yet fixed - s/calls_rejected/call_rejected/

Reply:
okay, fixed!